src/RVRP_data.jl

```diff
@@ -38,9 +42,11 @@ mutable struct Pickup
     id::String # If its part of a shipment, it has is own id anyway
     index::Int # Not given in JSON. If its part of a shipment, the index of the shipment
     location::Location
-    shipment_id::String # NULL if none
-    price_reward::Float64
-    capacity_request::Float64
+    shipment_id::String # NULL if pure Pickup request
+    product_id::String # optional : required only to implement conflicts
```

Comment:
If we decide to include the notion of commodoties (or products) we should have a separate structure for the commodity in order to not have redundant conflict descriptions in the JSON

Reply:
We need the `compartment_id`.
And I think it is linked to the commodity and not to the pickup/delivery